### PR TITLE
Updating docs to awso v2.10.0

### PR DESCRIPTION
**Pull request by akhil-sumologic**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)
https://sumologic.atlassian.net/browse/SUMO-246806

### CODE REVIEW

File: docs/observability/aws/deploy-use-aws-observability/before-you-deploy.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ Here’s a sample `sources.json` file that you can include in your AMI.
 
 Before setting up the AWS Observability solution we recommend testing permissions for both AWS and Sumo Logic by using a test AWS CloudFormation template. To execute this template:
 
-1. Invoke the AWS CloudFormation template at this [URL](https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/permissionchecker/permissioncheck.template.yaml).
+1. Invoke the AWS CloudFormation template at this [URL](https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/permissionchecker/permissioncheck.template.yaml).
```
Comment (jpipkin1) [inline docs/observability/aws/deploy-use-aws-observability/before-you-deploy.md:184]: This URL doesn't invoke the CloudFormation template. It only downloads the YAML file. You should get the download URL from the developers.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@akhil-sumologic, your changes to update docs to the new AWSO version look good. I found one thing you need to follow up on.

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ Given that we use an account alias, we recommend you use StackSets to automati
...
-    * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/sumologic_observability.master.template.yaml) to invoke the latest Sumo Logic AWS CloudFormation template.
-    * Download the AWS Observability Solution template (S3 Link for cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.
+    * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml) to invoke the latest Sumo Logic AWS CloudFormation template.
+    * Download the AWS Observability Solution template (S3 Link for cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:23]:
```suggestion
    * Download the AWS Observability Solution template (S3 link for CloudFormation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.
```

File: docs/observability/aws/deploy-use-aws-observability/update-aws-observability-stack.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Updating the Apps will create a backup of the old apps in a folder called as **
 To update an existing stack to the latest version:
 
 1. Locate the **Main Stack** created using AWS CloudFormation template and Click **Update**. <br/>  ![Stack_Step1.png](/img/observability/Stack_Step1.png)
-1. Select **Replace Current Template**, paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/sumologic_observability.master.template.yaml` in the Amazon S3 URL option, and then select **Next**.
+1. Select **Replace Current Template**, paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml` in the Amazon S3 URL option, and then select **Next**.
     :::note
     If you would like to download or inspect this or other versions of this template, please visit the [Change Log](changelog.md) page.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/update-aws-observability-stack.md:42]: Pls edit the last part of the sentence to read, "visit the [Changelog](changelog.md)."

**Review by kimsauce — COMMENTED** [review]
Had a couple formatting edits! Thanks for submitting @akhil-sumologic !

File: docs/observability/aws/deploy-use-aws-observability/before-you-deploy.md (hunk 1 above)
Comment (akhil-sumologic) [reply docs/observability/aws/deploy-use-aws-observability/before-you-deploy.md]: updated URL

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md (hunk 2 above)
Comment (akhil-sumologic) [reply docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md]: Addressed changes.

File: docs/observability/aws/deploy-use-aws-observability/update-aws-observability-stack.md (hunk 3 above)
Comment (akhil-sumologic) [reply docs/observability/aws/deploy-use-aws-observability/update-aws-observability-stack.md]: Done

File: docs/observability/aws/deploy-use-aws-observability/before-you-deploy.md (hunk 1 above)
Comment (jpipkin1) [reply docs/observability/aws/deploy-use-aws-observability/before-you-deploy.md]: Thank you!

**Review by jpipkin1 — APPROVED** [review]

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 4]
```diff
@@ -19,8 +19,8 @@ Given that we use an account alias, we recommend you use StackSets to automati
...
-    * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/sumologic_observability.master.template.yaml) to invoke the latest Sumo Logic AWS CloudFormation template.
-    * Download the AWS Observability Solution template (S3 Link for cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.
+    * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml) to invoke the latest Sumo Logic AWS CloudFormation template.
+    * Download the AWS Observability Solution template (S3 link for Cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:23]:
```suggestion
    * Download the AWS Observability Solution template (S3 link for CloudFormation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor casing fix

File: docs/observability/aws/deploy-use-aws-observability/changelog.md [hunk 5]
```diff
@@ -9,6 +9,15 @@ This section provides details on the available versions of the AWS Observability
 
 To install or upgrade to the required version of the CloudFormation template, use the URL of the YAML file corresponding to the right version number as the Amazon S3 URL in the AWS CloudFormation section of the AWS Management Console. See [Selecting a stack template](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-using-console-create-stack-template.html) for more information.
 
+## v2.10.0, 31-Aug-2024
+AWS Observability Solution (S3 Link for CloudFormation template): `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.10.0/sumologic_observability.master.template.yaml`
+
+AWS Observability Solution (Terraform-based) repo: [sumologic-solution-templates](https://github.com/SumoLogic/sumologic-solution-templates).
+
+Updates:
+* Added support for new sumo korea (kr) deployment.
```
Comment (JV0812) [inline docs/observability/aws/deploy-use-aws-observability/changelog.md:18]:
```suggestion
* Added support for new Sumo Logic Korea (kr) deployment.
```

File: docs/observability/aws/deploy-use-aws-observability/changelog.md [hunk 6]
```diff
@@ -9,6 +9,15 @@ This section provides details on the available versions of the AWS Observability
...
+* All python lambda runtimes have been updated to python3.12 along with libraries updated.
```
Comment (JV0812) [inline docs/observability/aws/deploy-use-aws-observability/changelog.md:19]:
```suggestion
* All python Lambda runtimes have been updated to python3.12 along with the required libraries.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes!

File: docs/observability/aws/deploy-use-aws-observability/changelog.md (hunk 6 above)
Comment (akhil-sumologic) [reply docs/observability/aws/deploy-use-aws-observability/changelog.md]: Done.

File: docs/observability/aws/deploy-use-aws-observability/changelog.md (hunk 5 above)
Comment (akhil-sumologic) [reply docs/observability/aws/deploy-use-aws-observability/changelog.md]: Done

**Review by JV0812 — APPROVED** [review]
LGTM

File: blog-service/2024-09-04-awso.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+keywords:
+  - AWSO
+  - observability
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We are excited to announce the release of AWS Observability 2.10.0. In this update we have updated the Lambda functions runtime version to Python 3.11 along with the required libraries.
```
Comment (JV0812) [inline blog-service/2024-09-04-awso.md:13]:
```suggestion
We are excited to announce the release of AWS Observability 2.10.0. In this update we have updated the Lambda functions runtime version to Python 3.12 along with the required libraries.
```

**Pull request merged**